src/winetricks

```diff
@@ -11971,6 +11971,60 @@ __EOF__
 
 #----------------------------------------------------------------
 
+w_metadata office2013pro apps \
+    title="Microsoft Office 2013 Professional" \
+    publisher="Microsoft" \
+    year="2013" \
+    media="cd" \
+    file1="setup.exe" \
+    installed_file1="$W_PROGRAMS_X86_WIN/Microsoft Office/Office15/WINWORD.EXE"
+
+load_office2013pro()
+{
+    if [ "$W_ARCH" = "win64" ]; then
+        w_die "Installation of Office 2013 Pro will not succeed on a 64bit wine prefix"
+    fi
+
+    if [ ! -x "$(which winbindd 2>/dev/null)" ]; then
+        w_warn "winbind is required"
+    fi
+
+    #link from https://www.askvg.com/direct-download-link-microsoft-office-2013-professional-plus-free-trial/
+    if ! test -f "$W_CACHE"/office2013pro/OFFICE15.iso; then
+        w_download http://care.dlservice.microsoft.com/dl/download/2/9/C/29CC45EF-4CDA-4710-9FB3-1489786570A1/OfficeProfessionalPlus_x86_en-us.img 236f8faae3f979ec72592a63784bba2f0d614916350c44631221b88ae9dae206 "OFFICE15.iso"
+    fi
+
+    w_set_winver win7
+
+    w_call corefonts
+    w_call msxml6
+    w_call riched20
+
+    # As of Wine 3.0, D3D11 works without this setting on Intel HD Graphics 510
+    if w_wine_version_in 3.0, ; then
```

Comment:
This should use w_workaround_wine_bug (with a linked bug report).

Reply:
I think it's [wine bug 38648](https://bugs.winehq.org/show_bug.cgi?id=38648)